webpack.config.js

```diff
@@ -38,7 +38,16 @@ module.exports = {
     ],
   },
   resolve: {
-    extensions: ['', '.js', '.jsx']
+    extensions: ['', '.js', '.jsx'],
```

Comment:
I'm doing all this with just this single line

```
root: [path.resolve(__dirname, 'src'), path.resolve(__dirname, 'node_modules')],
```

But I'm not sure wether it's better or not.